releasenotes/notes/lazy_sampling-93057adeaccbb46f.yaml

```diff
@@ -0,0 +1,22 @@
+---
+prelude: >
```

Comment:
This release note is a bit verbose. Can we move this info to our library docs and then link the docs in the release notes.

Reply:
I agree that the note is a bit verbose, and it's somewhat covered in the doc updates I did, still I think I'd prefer being verbose and informative in this case, and see little downside to the choice. However, if you feel strongly I can change.

Reply:
Actually thought about this a bit more and I added more to our actual docs around this, but going to keep the rn the same if that's alright.